torch/csrc/api/include/torch/nn/modules/any.h

```diff
@@ -422,16 +422,18 @@ AnyModule::AnyModule(std::shared_ptr<ModuleType> module)
           &std::remove_reference<ModuleType>::type::forward)) {
   // `AnyModule` can only store an `nn::Module` subclass object that provides
   // a `forward()` method that has a non-templatized return type.
-  // (e.g. `AnyModule` cannot store `nn::Sequential`, because `nn::Sequential`'s
-  // `forward()` method has a templatized return type.)
+  // (e.g. `AnyModule` cannot store `nn::AnySequential`, because
+  // `nn::AnySequential`'s `forward()` method has a templatized return type but
+  // you can store nn::Sequential instead.)
```

Comment:
I think here we should say "(... because `nn::AnySequential`'s `forward()` method has a templatized return type. However, `AnyModule` can store `nn::Sequential` because its `forward()` method has a non-templatized return type `Tensor`)"